src/python/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Mastho - 2020
+link_libraries(kdmp-parser)
+
+include_directories("C:\\Python36\\include")
```

Comment:
might need to do that differently as it needs to also compile on Linux - there might be a magic cmake module that finds those paths?